jetty-util/src/main/java/org/eclipse/jetty/util/ssl/SslContextFactory.java

```diff
@@ -353,35 +353,41 @@ public void dump(Appendable out, String indent) throws IOException
     
         try
         {
-            /* Use a pristine SSLEngine (not one from this SslContextFactory).
-             * This will allow for proper detection and identification
-             * of JRE/lib/security/java.security level disabled features
-             */
-            SSLEngine sslEngine = SSLContext.getDefault().createSSLEngine();
-    
-            List<Object> selections = new ArrayList<>();
-            
-            // protocols
-            selections.add(new SslSelectionDump("Protocol",
-                    sslEngine.getSupportedProtocols(),
-                    sslEngine.getEnabledProtocols(),
-                    getExcludeProtocols(),
-                    getIncludeProtocols()));
-            
-            // ciphers
-            selections.add(new SslSelectionDump("Cipher Suite",
-                    sslEngine.getSupportedCipherSuites(),
-                    sslEngine.getEnabledCipherSuites(),
-                    getExcludeCipherSuites(),
-                    getIncludeCipherSuites()));
-            
+            List<SslSelectionDump> selections = selectionDump();
             ContainerLifeCycle.dump(out, indent, selections);
         }
         catch (NoSuchAlgorithmException ignore)
         {
             LOG.ignore(ignore);
         }
     }
+
+    public List<SslSelectionDump> selectionDump() throws NoSuchAlgorithmException
```

Comment:
Not keen on having this method `public`, it's too much an implementation detail.
Can you make it package private?